chain/cmd/bandoracled2/handler.go

```diff
@@ -25,17 +26,25 @@ func handleTransaction(c *Context, l *Logger, tx tmtypes.TxResult) {
 	}
 
 	for _, log := range logs {
-		// TODO: Also handle IBC request packet here
-		messageType, err := GetEventValue(log, "message", "action")
+		messageType, err := GetEventValue(log, sdk.EventTypeMessage, otypes.AttributeKeyAction)
 		if err != nil {
 			l.Error(":cold_sweat: Failed to get message action type with error: %s", err.Error())
 			continue
 		}
-		if messageType != "request" {
-			l.Debug(":ghost: Skipping non-request message type: %s", messageType)
-			continue
+
+		if messageType == otypes.EventTypeRequest {
+			go handleRequestLog(c, l, log)
+		} else if messageType == (ibc.MsgPacket{}).Type() {
+			// Try to get request id from packet. If not then return error.
+			_, err := GetEventValue(log, otypes.EventTypeRequest, otypes.AttributeKeyID)
+			if err != nil {
+				l.Debug(":ghost: Skipping non-request packet type: %s", messageType)
```

Comment:
I don't think we need %s, messageType here. because it will always be ics/opaque